src/mcp_atlassian/models/jira.py

```diff
@@ -876,8 +901,8 @@ def to_simplified_dict(self) -> dict[str, Any]:
                     value = field_mapping[field]()
                     if value is not None:  # Only include non-None values
                         result[field] = value
-                # Handle custom fields
-                elif field.startswith("customfield_") and field in self.custom_fields:
+                # All other fields are custom fields
+                elif field in self.custom_fields:
```

Comment:
Good addition 👍. This ensures that fields captured into `self.custom_fields` during parsing can be correctly retrieved in the output when requested by name.